### ISSUE TITLE
Bug: Gradient not disabled/enabled in TensorQuantizer.disable_clip()/TensorQuantizer.enable_clip()

### ISSUE THREAD

**Issue by tapiomanninen** [NVIDIA/TensorRT#2608]
https://github.com/NVIDIA/TensorRT/blob/96e23978cd6e4a8fe869696d3d8ec2b47120629b/tools/pytorch-quantization/pytorch_quantization/nn/modules/tensor_quantizer.py#L160

Correct spelling for the pytorch variable to disable/enable gradient computation is "requires_grad", not "required_grad".

**Comment by zerollzeng** [comment]
@ttyio ^ ^

**Comment by ttyio** [comment]
thank you @tapiomanninen  for point out this, will fix!

**Comment by ttyio** [comment]
fixed in internal repo, will branch out to public soon, closing and thanks again!
